index.html

```diff
@@ -0,0 +1,277 @@
+        </svg>
+      </a>
+
+      <span class="span">Logout</span>
+      <span class="span">MyPage</span>
+      <span class="span">English</span>
+    </div>
+  </header>
+
+
+  <div class="container-fluid"></div>
+  <div class="navdiv sticky-top"></div>
+  <nav class="navbar d-none d-md-flex maxwidth ">
+    <a href="index.html"><img class="navbar-brand" src="./Images/Spectra-logo.webp" alt="Spectra-logo logo"> </a>
+    <div class="navlinks d-md-flex justify-content-between align-items-center">
+      <a href="#" class="active">Home</a>
+      <a href="about.html">About</a>
+      <a href="Services">Services</a>
+      <a href="#" class="nav-link disabled">Contact Us</a>
+      <button type="button">Book Now</button>
+    </div>
+  </nav>
+
+  <div class="menu">
+
+    <div id="myNav" class="overlay">
+
+      <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+
+      <div class="overlay-content">
+        <a href="#Home" onclick="closeNav()">Home</a>
+        <a href="about.html" onclick="closeNav()">About Us</a>
+        <a href="#" onclick="closeNav()">Services</a>
+        <a href="#" onclick="closeNav()">Contact Us</a>
+      </div>
+    </div>
+    <nav class="d-flex justify-content-between d-md-none maxwidth smallnav pl-3 navbar ">
+      <a href="index.html"><img class="navbar-brand-img1" src="./Images/Spectra-logo.webp" alt="Spectra logo"></a>
+      <span onclick="openNav()" class="fas fa-bars" data-toggle="collapse" data-target="#collapsibleNavbar"></span>
+      <div class="collapse navbar-collapse py-2" id="collapsibleNavbar">
+        <ul class="navbar-nav pl-3 float-right bg-light">
+          <li class="navlinks "><a href="index.html">Home</a></li>
+          <li class="navlinks"><a href="about.html">About Us</a></li>
+          <li class="navlinks"><a href="Services">Services</a></li>
+        </ul>
+      </div>
+    </nav>
+  </div>
+
+  <section class="intro">
+    <div class="intro-content">
+      <h1>Welcome to Spectra Beauté</h1>
+      <h2 class="intro-message">"We believe in our responsibility to inspire"</h2>
+    </div>
+    <div class="intro-desc">
+      <p>
+        A source of inspiration and well-being. When you hear of chic, professionalism, perfection, and outstanding
+        customer service, Spectra comes to mind. It is the number one leading beauty spa in Cameroon and Central Africa
+        as a whole. With all the chaos happening in our lives, Spectra is here to give you that peace and a relaxing
+        environment.
+      </p>
```

Comment:
- Kindly fix the overlay of this card for a better UI on the mobile screen 😉 
![screenshot (3)](https://user-images.githubusercontent.com/44624138/181944173-6b3e85b2-09be-43d0-a3fc-74d7588668ec.png)